{"uuid":"5fb7b494-111c-400a-ba63-5c6855d9b236","historyId":"f976741623b1eb29fbd1d21ff8344e06","fullName":"api.test.ApiTestMethod.executeTest","labels":[{"name":"package","value":"api.test.ApiTestMethod"},{"name":"testClass","value":"api.test.ApiTestMethod"},{"name":"testMethod","value":"executeTest"},{"name":"parentSuite","value":"Default Suite"},{"name":"suite","value":"Rest-Assured-testing-framework"},{"name":"subSuite","value":"project_save with priority 1"},{"name":"host","value":"localhost.localdomain"},{"name":"thread","value":"19086@localhost.localdomain.main(1)"},{"name":"framework","value":"testng"},{"name":"language","value":"java"}],"links":[],"name":"executeTest","status":"failed","statusDetails":{"known":false,"muted":false,"flaky":false,"message":"Assertion failed: project_save","trace":"java.lang.AssertionError: Assertion failed: project_save\n\tat org.testng.Assert.fail(Assert.java:87)\n\tat api.test.ApiTestMethod.executeTest(ApiTestMethod.java:70)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)\n\tat java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:77)\n\tat java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)\n\tat java.base/java.lang.reflect.Method.invoke(Method.java:569)\n\tat org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:133)\n\tat org.testng.internal.TestInvoker.invokeMethod(TestInvoker.java:598)\n\tat org.testng.internal.TestInvoker.invokeTestMethod(TestInvoker.java:173)\n\tat org.testng.internal.MethodRunner.runInSequence(MethodRunner.java:46)\n\tat org.testng.internal.TestInvoker$MethodInvocationAgent.invoke(TestInvoker.java:824)\n\tat org.testng.internal.TestInvoker.invokeTestMethods(TestInvoker.java:146)\n\tat org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:146)\n\tat org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:128)\n\tat java.base/java.util.ArrayList.forEach(ArrayList.java:1511)\n\tat org.testng.TestRunner.privateRun(TestRunner.java:794)\n\tat org.testng.TestRunner.run(TestRunner.java:596)\n\tat org.testng.SuiteRunner.runTest(SuiteRunner.java:377)\n\tat org.testng.SuiteRunner.runSequentially(SuiteRunner.java:371)\n\tat org.testng.SuiteRunner.privateRun(SuiteRunner.java:332)\n\tat org.testng.SuiteRunner.run(SuiteRunner.java:276)\n\tat org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:53)\n\tat org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:96)\n\tat org.testng.TestNG.runSuitesSequentially(TestNG.java:1212)\n\tat org.testng.TestNG.runSuitesLocally(TestNG.java:1134)\n\tat org.testng.TestNG.runSuites(TestNG.java:1063)\n\tat org.testng.TestNG.run(TestNG.java:1031)\n\tat com.intellij.rt.testng.IDEARemoteTestNG.run(IDEARemoteTestNG.java:66)\n\tat com.intellij.rt.testng.RemoteTestNGStarter.main(RemoteTestNGStarter.java:109)\nCaused by: java.lang.AssertionError: users.firstName\nExpected: thusitha\n     got: Kavinda\n ; users.lastName\nExpected: dinesh\n     got: Medagoda\n\n\tat org.skyscreamer.jsonassert.JSONAssert.assertEquals(JSONAssert.java:417)\n\tat org.skyscreamer.jsonassert.JSONAssert.assertEquals(JSONAssert.java:394)\n\tat org.skyscreamer.jsonassert.JSONAssert.assertEquals(JSONAssert.java:336)\n\tat api.test.ApiTestMethod.executeTest(ApiTestMethod.java:59)\n\t... 27 more\n"},"stage":"finished","steps":[{"name":"Test failed: users.firstName\nExpected: thusitha\n     got: Kavinda\n ; users.lastName\nExpected: dinesh\n     got: Medagoda\n","status":"passed","stage":"finished","steps":[],"attachments":[],"parameters":[],"start":1724168014659,"stop":1724168014660}],"attachments":[],"parameters":[],"start":1724168009539,"stop":1724168014670}